hawkular-alerts-actions-plugins/hawkular-alerts-actions-email/src/main/java/org/hawkular/alerts/actions/email/EmailTemplate.java

```diff
@@ -268,25 +264,27 @@ private void subjects(Map<String, Object> props, Alert alert) {
     }
 
     private String oneCondAvailabilityDescription(AvailabilityCondition cond) {
-        if (cond.getOperator().equals(AvailabilityCondition.Operator.UP)) {
-            return "up";
-        } else if (cond.getOperator().equals(AvailabilityCondition.Operator.NOT_UP)) {
-            return "not up";
-        } else if (cond.getOperator().equals(AvailabilityCondition.Operator.DOWN)) {
-            return "down";
+        switch (cond.getOperator()) {
+            case UP:
+                return "up";
+            case NOT_UP:
+                return "not up";
+            case DOWN:
+                return "down";
```

Comment:
I like the change to switch statement but we should probably move these text translations into the enum and just call cond.getOperator.asText() or something like that.